client/my-sites/stats/stats-main/index.js

```diff
@@ -0,0 +1,39 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect }	from 'react-redux';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { shouldViewBeVisible } from 'state/first-view/selectors';
+
+const StatsMain = React.createClass( {
+	displayName: 'StatsMain',
```

Comment:
`displayName` shouldn't be necessary, as we're assigning the class directly to the `StatsMain` constant. Also, since this is a render-only component, why not just define it as a pure function?

```js
const StatsMain = ( { isFirstViewVisible, children } ) => {
  const classes = classnames( … );
  return (
    <Main … >
      …
    </Main>
  );
};
```